data/items/items.xml

```diff
@@ -17590,6 +17590,7 @@
 	</item>
 	<item id="9975" article="a" name="crystal table">
 		<attribute key="weight" value="2500" />
+		<attribute key="wrapTo" value="9974" />
```

Comment:
@Zbizu Wrap/Unwrap are the same bytes, you can just do a table with ids = {[x] = y}, kind off already is written in construction kits
@Kamenuvol because items that can be wrapped is a type not an attribute, its more easier to deal with it in lua and makes more sense, since we got more similar itemtypes such as isTeleport and isTile, why not isWrappable.